### ISSUE TITLE
Improve Queue Page Accessibility

### ISSUE THREAD

**Issue by bhuvy2** [illinois/queue#248]
**Problem**
The settings page on the queue isn't accessible. Almost all of the section titles are missing, making it hard for screen readers and navigators to go between content

<img width="960" alt="settings_acc" src="https://user-images.githubusercontent.com/6509868/55023403-ba5a9f00-4fca-11e9-9b13-26b0d1c6124d.PNG">

**Proposed Solution**
* Add appropriate titles (too many to list)
